optimism/src/mips/interpreter.rs

```diff
@@ -243,7 +249,17 @@ pub fn interpret_itype<Env: InterpreterEnv>(env: &mut Env, instr: ITypeInstructi
         ITypeInstruction::LoadUpperImmediate => (),
         ITypeInstruction::Load8 => (),
         ITypeInstruction::Load16 => (),
-        ITypeInstruction::Load32 => (),
+        ITypeInstruction::Load32 => {
+            let rt = env.get_instruction_part(InstructionPart::RT);
+            // Values are the first 16bits
```

Comment:
First? Do you mean it is given in little endian? ([see](https://max.cs.kzoo.edu/cs230/Resources/MIPS/MachineXL/InstructionFormats.html), where the immediate value seems to be the "last" 16 bits)

Reply:
I change in the next PR.